<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenclient</class>
 <widget class="QDialog" name="fenclient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../client-build-desktop-Qt_4_8_1_for_Desktop_-_MSVC2010__Qt_SDK__Release/release/res/chat.png</normaloff>../client-build-desktop-Qt_4_8_1_for_Desktop_-_MSVC2010__Qt_SDK__Release/release/res/chat.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>191</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="nombreClientsLabel">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>1</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>En ligne (0)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>0</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../client-build-desktop-Qt_4_8_1_for_Desktop_-_MSVC2010__Qt_SDK__Release/release/res/enligne.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listeClients">
       <property name="minimumSize">
        <size>
         <width>191</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>191</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="boutonDeconnexion">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Deconnexion</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../client-build-desktop-Qt_4_8_1_for_Desktop_-_MSVC2010__Qt_SDK__Release/release/res/deconnect.png</normaloff>../client-build-desktop-Qt_4_8_1_for_Desktop_-_MSVC2010__Qt_SDK__Release/release/res/deconnect.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <widget class="QLineEdit" name="messageEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="boutonEmo">
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Emoticones&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../client-build-desktop-Qt_4_8_1_for_Desktop_-_MSVC2010__Qt_SDK__Release/release/res/smile.png</normaloff>../client-build-desktop-Qt_4_8_1_for_Desktop_-_MSVC2010__Qt_SDK__Release/release/res/smile.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="boutonCouleur">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Couleur du texte&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../client-build-desktop-Qt_4_8_1_for_Desktop_-_MSVC2010__Qt_SDK__Release/release/res/color.png</normaloff>../client-build-desktop-Qt_4_8_1_for_Desktop_-_MSVC2010__Qt_SDK__Release/release/res/color.png</iconset>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="boutonEnvoyer">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>Envoyer</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../client-build-desktop-Qt_4_8_1_for_Desktop_-_MSVC2010__Qt_SDK__Release/release/res/envoyer.png</normaloff>../client-build-desktop-Qt_4_8_1_for_Desktop_-_MSVC2010__Qt_SDK__Release/release/res/envoyer.png</iconset>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>460</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="icon">
         <iconset>
          <normaloff>../client-build-desktop-Qt_4_8_1_for_Desktop_-_MSVC2010__Qt_SDK__Release/release/res/public.png</normaloff>../client-build-desktop-Qt_4_8_1_for_Desktop_-_MSVC2010__Qt_SDK__Release/release/res/public.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Public</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextEdit" name="listeMessages">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="autoFormatting">
            <set>QTextEdit::AutoNone</set>
           </property>
           <property name="tabChangesFocus">
            <bool>false</bool>
           </property>
           <property name="undoRedoEnabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::NoTextInteraction</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
